src/crypto/randomx/reciprocal.c

```diff
@@ -52,10 +52,7 @@ uint64_t randomx_reciprocal(uint64_t divisor) {
 
 	uint64_t quotient = p2exp63 / divisor, remainder = p2exp63 % divisor;
 
-	unsigned bsr = 0; //highest set bit in divisor
-
-	for (uint64_t bit = divisor; bit > 0; bit >>= 1)
-		bsr++;
+	unsigned bsr = 64 - __builtin_clzll(divisor); //highest set bit in divisor
```

Comment:
This will not compile on MSVC, and it's not a performance-critical part of the code, so it's better to return the previous code here.